array/models/structures/StructContact.model.bxb

```diff
@@ -0,0 +1,11 @@
+structure (StructContact) {
```

Comment:
Let's just call this `Contact`. Usually we do not reflect the type in the concept name.

Reply:
Naming conversion, this is the part we need to discuss.